spec/support/record_shared.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
-describe GapIntelligence::Record do
-  it 'allows records to be created' do
+shared_examples_for 'Record' do
```

Comment:
@gtimti glad that we are on the same page with that idea itself, I used Category one was just to introduce that new way, will do for sure